server/controllers/tripController.js

```diff
@@ -0,0 +1,44 @@
+import tripHelpers from '../helpers/tripHelpers';
+
+/**
+ * This class contains all methods
+ * required to handle
+ * trip endpoints' request.
+ */
+class TripController {
+  /**
+   * This method handle oneWayTrip request.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the trip.
+   */
+  static async oneWayTrip(req, res) {
```

Comment:
1. this function should n't be called oneWay, as it should be used even for return and multicity. 
2. you have two options for the trip type, either to enter data and you define the type by your controller based on the data. or you choose the trip type at the user level and you create a validator.

Reply:
yes this is the good way to do this thank you for this good idea